chain/actors/builtin/verifreg/actor.go.template

```diff
@@ -20,14 +20,58 @@ import (
 	"github.com/filecoin-project/lotus/chain/actors/adt"
 	"github.com/filecoin-project/lotus/chain/actors"
 	"github.com/filecoin-project/lotus/chain/types"
-    verifregtypes "github.com/filecoin-project/go-state-types/builtin/v9/verifreg"
+	verifregtypes "github.com/filecoin-project/go-state-types/builtin/v9/verifreg"
+	verifregtypes13 "github.com/filecoin-project/go-state-types/builtin/v14/verifreg"
```

Comment:
should be 12 .. not 13 or 14, and I'm not sure how I ended up with those numbers :chinscratch:

`MinimumVerifiedAllocationTerm` is the main thing we need, it only showed up in 12; we can't bump everything to 12 without breaking the entire `verifreg.State` API which requires v9 properties, most notably `AllocationId`. So either we break them all to require v12 properties, or import both just to expose a stable `MinimumVerifiedAllocationTerm`.

I've removed the rest of the properties, I was just trying to be complete instead of just the random grab-bag of properties currently exposed.